src/diffpy/srfit/sas/prcalculator.py

```diff
@@ -92,7 +92,7 @@ def __call__(self, r):
         self._invertor.y = iq
         self._invertor.err = diq
         c, c_cov = self._invertor.invert_optimize()
-        calculate_pr = lambda x: self._invertor.pr(c, x)
+        calculate_pr = lambda x: self._invertor.pr(c, x)  # noqa: E731
```

Comment:
This one also better to put it in the map directly?  Or maybe as an explicit function